manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer.yaml

```diff
@@ -77,22 +81,26 @@ spec:
           {{- if .Values.linux.privileged }}
             privileged: true
           {{- end }}
+          {{- if .Values.enableArcExtension }}
           {{- if .Values.Azure.proxySettings.isProxyEnabled }}
```

Comment:
Could we just and here instead of 2 ifs?

Reply:
I have moved arc specific values to `arc-values.yaml` Because of which this lines gives error saying it can't find `Values.Azure.proxySettings.isProxyEnabled` 
`.Values.enableArcExtension` condition is the way to avoid these errors and hence introduce to gate monitoring resources.

Reply:
sounds good!